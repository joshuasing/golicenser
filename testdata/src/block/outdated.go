/*
Copyright (c) 2001 Test
*/

package block // want "invalid license header"
