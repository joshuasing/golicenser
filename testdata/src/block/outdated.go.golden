/*
Copyright (c) 2025 Test
*/

package block // want "invalid license header"
